pycbc/sensitivity.py

```diff
@@ -164,49 +168,43 @@ def volume_montecarlo(found_d, missed_d, found_mchirp, missed_mchirp,
         if max_param is not None:
             max_distance = max_param
         else:
-            # if no max distance given, use max distance actually injected
+            # If no max distance given, use max distance actually injected
             max_distance = max(found_d.max(), missed_d.max())
     else:
         raise NotImplementedError("%s is not a recognized parameter"
                                   % limits_param)
 
-    # volume of sphere
+    # Calculate volume of sphere out to maximum distance
     montecarlo_vtot = (4. / 3.) * numpy.pi * max_distance**3.
 
-    # arrays of weights for the MC integral
+    # Calculate weights based on distribution parameters
     if distribution_param == 'distance':
-        found_weights = found_d ** d_power
-        missed_weights = missed_d ** d_power
+        found_weights = numpy.power(found_d, d_power)
```

Comment:
I find this in the documentation for numpy power: 
`The ** operator can be used as a shorthand for np.power on ndarrays`

Hence, I do not believe this change to use numpy.power can have any effect on performance, and it reduces readability IMO. Please revert these specific changes unless they are shown to significantly improve performance.